### DISCUSSION TITLE
How to get all ASINs for products sold by a specific Seller

### DISCUSSION THREAD

**Discussion by RobinPlanning** [amzn/selling-partner-api-models#3676] [General]
I've tried using "searchCatalogItems" to retrieve all ASINs for a specific SellerID, but that didn't work.
I'm thinking of using the "GET_MERCHANT_LISTINGS_ALL_DATA" report, but I can get all my own ASINs, but not the product list for any SellerID.
Is there any way I can help?

**Comment by vikingcodes** [comment]
@RobinPlanning I can help you to get all ASINs of specific seller.
